models/namespace.go

```diff
@@ -16,3 +16,8 @@ type NamespaceSpec struct {
 }
 
 const AllNamespace = "*"
+
+type NamespaceJobNamePair struct {
```

Comment:
we don't need this struct, we can avoid this.

Reply:
removed this as we are no longer need namespace to refresh specific jobs.